presto-docs/src/main/sphinx/presto-cpp.rst

```diff
@@ -0,0 +1,55 @@
+**********
+Presto C++
+**********
+
+Note: Presto C++ is in active development. See :doc:`Limitations </presto_cpp/limitations>`.
+
+.. toctree::
+    :maxdepth: 1
+
+    presto_cpp/features
+    presto_cpp/limitations
+    presto_cpp/properties
+
+Overview
+========
+
+Presto C++, sometimes referred to by the development name Prestissimo, is a C++ 
+drop-in replacement for Presto workers based on the `Velox <https://velox-lib.io/>`_ library. It 
```

Comment:
C++  drop-in replacement for Presto workers based on the

-->

drop-in replacement for Presto workers written in C++ and based on the